### ISSUE TITLE
Dropdown accessibility improvements

### ISSUE THREAD

**Issue by desig9stein** [IgniteUI/igniteui-webcomponents#475]
## Description  
The screen reader does not read any of the items inside the dropdown.

**Comment by desig9stein** [comment]
Still reproducible: When you open the dropdown, it reads all the item values, including the headers if available. However, it doesn't tell you which item is selected or which one you are currently on.

**Comment by simeonoff** [comment]
@desig9stein Seems this issue cannot be fixed until there is some mechanism that allows us to reflect the aria-related attributes from within the shadow DOM into the light DOM.

There are several ongoing proposals for how this can be resolved, but I believe the latest one can be found [here](https://github.com/WICG/webcomponents/issues/917).
